lib/tools.js

```diff
@@ -91,8 +91,7 @@ function genLog (z) {
     if (p > 1) {
       l = typeof a === 'string' ? countInterp(a, '%j') : 0
       if (l) {
-        n.length = l + countInterp(a, '%d') + countInterp(a, '%s') + 1
-        o = `${util.format.apply(null, n)}`
+        o = util.format.apply(null, n)
```

Comment:
the ES6 string is deliberate - (pretty much every oddity in this code is deliberate) - it triggers an optimization (can't remember what, but this is (or was) a technique is being used in core)

to determine whether we can remove the template string optimization we'll need benchmarks for 4, 6, and 8 *just* for the removal of the back ticks - can we put the back ticks back in for this PR and if you're up for it we can look at the backtick removal  in a separate a PR with it's own benchmarks

Reply:
Sure, I'll just remove it from this PR and figure whether or to remove it separately.